server/graphql/resolvers.ts

```diff
@@ -28,6 +28,10 @@ const resolvers = {
     }),
     clients: (_, __, { dataSources, authenticateUser }): Array<ClientInterface> => authenticateUser().then(() => dataSources.clients.getAll()),
     request: (_, { id }, { dataSources }): RequestInterface => dataSources.requests.getById(Types.ObjectId(id)),
+    getClientRequests: (_, {firstName}, {dataSources}): Array<RequestInterface> => dataSources.requests.getAll().filter(request => {
```

Comment:
I'm curious about why we're only getting clients by first name

Reply:
oh in the task description it said to fetch all of a client's requests given their first name so I just followed it